Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -105,18 +115,26 @@ int main()
         // notifications to the sample
         case ExtendedActivationKind::Launch:
         {
-            // Request a WNS Channel URI which can be passed off to an external app to send notifications to.
-            // The WNS Channel URI uniquely identifies this app for this user and device.
-            PushNotificationChannel channel{ RequestChannel() };
-            if (!channel)
```

Comment:
I think this pattern for null is incorrect. You should actually check for the Extended error here. Can you please sync with @sharath2727 on the correct pattern?